src/api/impl/ConnectedServer.hh

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "ServerBase.hh"
+
+namespace ra {
+namespace api {
+namespace impl {
+
+class ConnectedServer : public ServerBase
+{
+public:
+    ConnectedServer(const std::string& sHost) : m_sHost(sHost) {}
```

Comment:
You should put the `explicit`. `noexcept` probably wouldn't work here since the string is copied.